### DISCUSSION TITLE
Avalonia with .net 6

### DISCUSSION THREAD

**Discussion by llfab** [AvaloniaUI/Avalonia#7024] [General]
Just wanted to let everyone know what sort of worked for me while waiting for an official version of AvaloniaVS for VS2022:
- We have converted our source code from .net 5 to .net 6
- This requires (officially) VS 2022 to let .net 6 work 100% properly but then there is no AvaloniaVS for VS2022 and therefore there would not be any templates and designer in VS 2022
- As an intermediate solution we have made sure that all project files use only features that are compatible with VS2019. As an outcome we can open the solution in both VS2019 and 2022 with .net 6. It builds and run without the AvaloniaVS 2022 in VS2022. If I need the designer, I open the solution in VS2019 where it builds with some few warnings about .net6 not being meant for VS 2019 but it also works

This closes a current gap for us while waiting until AvaloniaVS is available for VS2022. 

Any ideas about the timing of this?

**Comment by rstm-sf** [comment]
In general, a Rider with net 6 works well. .NET supports via Resharper. And accordingly, many Resharper features are present, such as better support for Avalonia
